apps/i18n/common/en_us.json

```diff
@@ -1598,6 +1598,10 @@
   "ltiSectionSyncButtonDisabledAltText": "Roster sync is disabled. To re-enable, go to your account settings.",
   "ltiSectionSyncDialogDescription": "New to Code.org? A [section]({aboutSectionsUrl}) is used to manage your classroom and assign curricula. We've identified the following sections & students associated with your course.\n\nYou can re-sync your sections & students at any time! Learn more [about syncing here]({aboutSyncingUrl}).",
   "ltiSectionSyncDialogError": "We encountered an error when trying to sync with your LMS.",
+  "ltiSectionSyncDialogErrorNoIntegration": "LTI Integration not found",
+  "ltiSectionSyncDialogErrorNoSectionFound": "We couldn't find the given section.",
+  "ltiSectionSyncDialogErrorWrongContext": "Sorry, looks like you are attempting to sync a course or section from the wrong place. Please launch the Code.org Integration from inside the course or section you would like to sync.",
+  "ltiSectionSyncDialogErrorSchoologyDetails": "Read more about supported Schoology placements [here](%{url}).",
```

Comment:
We don't want to hardcode to Schoology, rather the LMS name of whatever LMS Platform they are on.

Reply:
Alternatively, we could just remove `Schoology` here, e.g. `Read more about supported placements here`

Reply:
Yeah we could have one support page with placement lists for all supported platforms

Reply:
Sorry, I see now that this string key is Schoology specific, but I still don't think it needs to be? We can either make it generic (by removing `Schoology` as mentioned), and still can interpolate the specific URL, _or_ interpolate the issuer name here as well.